src/main/java/co/org/osso/api/BookService.java

```diff
@@ -0,0 +1,52 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class BookService {
+
+    Map<String, Book> mapBooks = new HashMap<>();
+    int id;
+
+    public Book createBook(Book book) {
+        book.setBookID(""+id++);
+        mapBooks.put(book.getBookID(), book);
+        return book;
+    }
+
+    public Book getBook(String id) {
+        if (mapBooks.get(id) != null){
+            return mapBooks.get(id);
+        } else {
+            return new Book();
+        }
+    }
+}
+
+class Book{
+    String bookID;
+    String title;
+    String year;
+
+    public void setBookID(String ID){
+        this.bookID = ID;
+    }
+
+    public void setTitle(String title){
+        this.title = title;
+    }
+
+    public void setYear(String year){
+        this.year = year;
+    }
+
+    public String getBookID(){ return bookID; }
```

Comment:
I don't mind this format for simple methods like getters and setters, but I think we should be consistent within the class, and not use both the 3-line version and the 1-line version. Choose one of them, and stick to it.

Reply:
Ok, I understand. Done